### ISSUE TITLE
UnicodeDecodeError: 'utf8' codec can't decode bytes when using Stanford Parser

### ISSUE THREAD

**Issue by HikaruL** [nltk/nltk#1770]
This is my original codes, I have declare # coding: utf8 on top of the script, and I'm using the latest stanford nlp package. 
Some sentences can be parsed successfully, but most cannot. I also tried adding encoding='utf8' as parameter in the parser and got the same error. Not sure how to solve it. thanks!

![parser](https://user-images.githubusercontent.com/19523304/27877187-8f5991ae-616e-11e7-8fa8-8a5ed70a621f.jpeg)
![error](https://user-images.githubusercontent.com/19523304/27877190-93a28928-616e-11e7-80f9-e565f0cf7734.jpeg)

**Comment by HikaruL** [comment]
segmenter works fine

**Comment by alvations** [comment]
@HikaruL, on your terminal, **what is your output for `locale`?**, e.g.:

```
$ locale
LANG="en_us.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_CTYPE="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_ALL="en_US.UTF-8"
```

If you don't see `LC_ALL="en_US.UTF-8"`, then on the terminal:

```
export LC_ALL=en_US.UTF-8
```

Did the locale trick work? 

If so, set the locale permanently, e.g.:

```
dpkg-reconfigure locales
localedef -i en_US -c -f UTF-8 en_US.UTF-8
```

then restart the machine and you should see `LC_ALL="en_US.UTF-8"`.

**Comment by HikaruL** [comment]
I tried setting LC_ALL and it still gave me the same error

**Comment by alvations** [comment]
What is your output for `$ locale` on the terminal?

**Comment by HikaruL** [comment]
![locale](https://user-images.githubusercontent.com/19523304/27897966-45761dd6-61d8-11e7-85b8-337c8d23d611.jpeg)

**Comment by alvations** [comment]
Try:

```
export LC_CTYPE=en_US.utf-8
```

If it works, add the line above to your `~/.bash_profile`

**Comment by alvations** [comment]
**What is your output for this?**:

```
$ python
Python 2.7.13 (default, Dec 18 2016, 07:03:39) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> print sys.stdout.encoding
UTF-8
```

Please avoid posting images, post the code instead. That'll help us to help you better =)

----

P/S: Also, avoid the old Stanford API, use the new CoreNLP APIs instead, see https://github.com/nltk/nltk/pull/1735#issuecomment-306137326

**Comment by alvations** [comment]
@HikaruL are you still facing the same problem? 

BTW, what is your output of `sys.stdout.encoding` in your python environment? That'll help us debug what went wrong.

**Comment by alvations** [comment]
This should be fixed with the #1735 in NLTK v3.2.5 with the deprecation of the old API.
And the Stanford API would be further improved with v3.2.6 =)

**Comment by stevenbird** [comment]
Thanks @alvations
